src/Common/tests/System/RuntimeDetection.cs

```diff
@@ -12,7 +12,7 @@ public static class RuntimeDetection
         private static readonly string s_frameworkDescription = RuntimeInformation.FrameworkDescription;
 
         // public static bool IsMono { get; } = Type.GetType("Mono.Runtime") != null;
-        // public static bool IsNetFramework { get; } = s_frameworkDescription.StartsWith(".NET Framework");
+        public static bool IsNetFramework { get; } = s_frameworkDescription.StartsWith(".NET Framework");
```

Comment:
never-mind I see you use it below but I guess I'm not sure why it was ever commented out.